modules/dockermodelrunner/main_test.go

```diff
@@ -0,0 +1,34 @@
+package dockermodelrunner_test
+
+import (
+	"context"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
suggestion: this won't show the tests as skipped, use a helper include a once if needed.

Reply:
The issue is with the testable examples, that cannot use t.Skip. I can provide a helper function that checks that, but then if not on Docker Desktop, I need to printout the expected outputs in that early code branch.

Not against this, though